backend/src/models/user/postgres.js

```diff
@@ -65,6 +65,8 @@ const UserRepo = (postgres) => {
       return err;
     }
   };
+  const checkUserSQL = `
```

Comment:
There is already a SQL command for this getUserInfoByEmailSQL

Reply:
Thanks I fixed this